packages/dashboard/src/stores/client-store.ts

```diff
@@ -0,0 +1,19 @@
+import { action, observable } from 'mobx';
+import { TClientProps } from 'Types';
+import BaseStore from './base-store';
+
+class ClientStore extends BaseStore {
```

Comment:
why are we declaring new stores for client in the `dashboard` package?
is it possible to use these stores from the `core` package?

Reply:
We're running the package in both Deriv and Binary Static, so there won't be any client store in Binary Static.